clustered/client/src/main/java/org/ehcache/clustered/client/config/builders/ClusteringServiceConfigurationBuilder.java

```diff
@@ -76,7 +68,39 @@ public ServerSideConfigurationBuilder autoCreate() {
    * @return a clustering service configuration builder
    */
   public ServerSideConfigurationBuilder expecting() {
-    return new ServerSideConfigurationBuilder(new ClusteringServiceConfigurationBuilder(this, false));
+    return new ServerSideConfigurationBuilder(new ClusteringServiceConfigurationBuilder(this.clusterUri, this.timeouts, false));
+  }
+
+  /**
+   * Adds timeouts.
+   * Read operations which time out return a result comparable to a cache miss.
+   * Mutative operations which time out won't do anything.
+   * Lifecycle operations which time out will fail with exception
+   *
+   * @param timeouts the amount of time permitted for all operations
+   *
+   * @return a clustering service configuration builder
+   *
+   * @throws NullPointerException if {@code timeouts} is {@code null}
+   */
+  public ClusteringServiceConfigurationBuilder timeouts(Timeouts timeouts) {
+    return new ClusteringServiceConfigurationBuilder(this.clusterUri, timeouts, this.autoCreate);
+  }
+
+  /**
+   * Adds timeouts.
+   * Read operations which time out return a result comparable to a cache miss.
+   * Mutative operations which time out won't do anything.
+   * Lifecycle operations which time out will fail with exception
+   *
+   * @param timeoutsBuilder the builder for amount of time permitted for all operations
+   *
+   * @return a clustering service configuration builder
+   *
+   * @throws NullPointerException if {@code timeouts} is {@code null}
+   */
+  public ClusteringServiceConfigurationBuilder timeouts(Timeouts.Builder timeoutsBuilder) {
```

Comment:
The parameter here should really be `Builder<? extends Timeouts>`.

Reply:
Right